src/amount.h

```diff
@@ -15,6 +15,8 @@ typedef int64_t CAmount;
 
 static const CAmount COIN = 100000000;
 static const CAmount CENT = 1000000;
+static const size_t KB = 1000;
+static const size_t PRECISION_MULTIPLIER = KB;
```

Comment:
Then we agree that it's confusing. Please do rename it for this pull.
```
Unless I'm missing something, such a constant would need to be initialized in the cpp. That is, for example, what I dislike about
```
Not true, try to compile the following:
```c++
#include <stdio.h>
class TestClass {
public:
    static const int SubValue = 12345;
};
int main()
{
    printf("%i\n", TestClass::SubValue);
}
```
*String constants* and other objects need to be initialized in the cpp, no matter where they're defined. But *integral types* (such as ints) can be initialized in the header.